Build failure: virtualgl-lib

### Nixpkgs version

- Unstable (25.11)

### Steps to reproduce

nix build github:NixOS/nixpkgs/d3baaf296366efdea6737124f05b65d1cf25fa7c#virtualglLib

### Can Hydra reproduce this build failure?

Hydra is currently rebuilding this package.

### Link to Hydra build job

_No response_

### Relevant log output

```console
Running phase: unpackPhase
@nix { "action": "setPhase", "phase": "unpackPhase" }
unpacking source archive /nix/store/96q22yfv862n070wxygnbdjyjkqmdlsw-VirtualGL-3.0.2.tar.gz
source root is VirtualGL-3.0.2
setting SOURCE_DATE_EPOCH to timestamp 1666223892 of file "VirtualGL-3.0.2/wgldemos/wglspheres.c"
Running phase: patchPhase
@nix { "action": "setPhase", "phase": "patchPhase" }
Running phase: updateAutotoolsGnuConfigScriptsPhase
@nix { "action": "setPhase", "phase": "updateAutotoolsGnuConfigScriptsPhase" }
Running phase: configurePhase
@nix { "action": "setPhase", "phase": "configurePhase" }
cmake flags: -DCMAKE_FIND_USE_SYSTEM_PACKAGE_REGISTRY=OFF -DCMAKE_FIND_USE_PACKAGE_REGISTRY=OFF -DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON -DCMAKE_BUILD_TYPE=Release -DBUILD_TESTING=OFF -DCMAKE_INSTALL_LOCALEDIR=/nix/store/hyjcd25ia73jhfn2lmqkxkpgg6b>
CMake Error at CMakeLists.txt:5 (cmake_minimum_required):
  Compatibility with CMake < 3.5 has been removed from CMake.

  Update the VERSION argument <min> value.  Or, use the <min>...<max> syntax
  to tell CMake that the project requires at least <min> but has been updated
  to work with policies introduced by <max> or earlier.

  Or, add -DCMAKE_POLICY_VERSION_MINIMUM=3.5 to try configuring anyway.


-- Configuring incomplete, errors occurred!
```

### Additional context

Package was build with current nixos-unstable-small branch (10/6/2025, 6:35:55 PM GMT+2)
Seems to be a similar issue to https://github.com/NixOS/nixpkgs/issues/448918 and https://github.com/NixOS/nixpkgs/issues/448304
but the PR in the later issue does not fix the build failure for virtualgl-lib

### System metadata

- system: `"x86_64-linux"`
 - host os: `Linux 6.17.0, NixOS, 25.11 (Xantusia), 25.11.20251004.6c4f138`
 - multi-user?: `yes`
 - sandbox: `yes`
 - version: `nix-env (Nix) 2.28.5`
 - nixpkgs: `/nix/store/hq5l53mw27n5mm2hjpiri1iac9h377ih-source`

### Notify maintainers


@abbradar 
---

**Note for maintainers:** Please tag this issue in your pull request description. (i.e. `Resolves #ISSUE`.)


### I assert that this issue is relevant for Nixpkgs

- [x] I assert that this is a bug and not a support request.
- [x] I assert that this is not a [duplicate of an existing issue](https://github.com/NixOS/nixpkgs/issues?q=is%3Aopen+is%3Aissue+label%3A%220.kind%3A+build+failure%22). 
- [x] I assert that I have read the [NixOS Code of Conduct](https://github.com/NixOS/.github/blob/master/CODE_OF_CONDUCT.md) and agree to abide by it.

### Is this issue important to you?

Add a :+1: [reaction] to [issues you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[issues you find important]: https://github.com/NixOS/nixpkgs/issues?q=is%3Aissue+is%3Aopen+sort%3Areactions-%2B1-desc

--- a/pkgs/tools/X11/virtualgl/lib.nix
+++ b/pkgs/tools/X11/virtualgl/lib.nix
@@ -35,6 +35,7 @@ stdenv.mkDerivation rec {
   cmakeFlags = [
     "-DVGL_SYSTEMFLTK=1"
     "-DTJPEG_LIBRARY=${libjpeg_turbo.out}/lib/libturbojpeg.so"
+    (lib.cmakeFeature "CMAKE_POLICY_VERSION_MINIMUM" "4.0")
   ];
 
   makeFlags = [ "PREFIX=$(out)" ];

